examples/send_tx.rs

```diff
@@ -0,0 +1,70 @@
+use near_jsonrpc_client::{methods, JsonRpcClient};
+use near_jsonrpc_primitives::types::query::QueryResponseKind;
+use near_primitives::transaction::{Action, FunctionCallAction, Transaction};
+use near_primitives::types::BlockReference;
+
+use serde_json::json;
+
+mod utils;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    env_logger::init();
+
+    let client = JsonRpcClient::connect("https://rpc.testnet.near.org");
+
+    let signer_account_id = utils::input("Enter the signer Account ID: ")?.parse()?;
+    let signer_secret_key = utils::input("Enter the signer's private key: ")?.parse()?;
+    let wait_until_str = utils::input("Enter the desired guaranteed execution status: ")?;
+    let wait_until = serde_json::from_str(&("\"".to_owned() + &wait_until_str + "\""))?;
```

Comment:
Unfortunately, it's not true.
I also need to put my str into quotes to create a valid json. With adding quotes, your option is the same as mine.

Reply:
@telezhnaya Are you sure you tried `from_value` (not `from_str`) and pass `json!(value)` instead of String value? This code should work as far as I can tell.

Reply:
`value` should be a valid json in `json!(value)`. If it's string, it should be quoted.

Reply:
@telezhnaya This is what we meant here: https://github.com/near/near-jsonrpc-client-rs/pull/136/commits/a4f27a4a6155b5572876dd46950ec8d3a680f12e